src/openrct2-ui/windows/RideList.cpp

```diff
@@ -206,14 +206,6 @@ namespace OpenRCT2::Ui::Windows
                 Invalidate();
                 height = min_height;
             }
-
-            // Refreshing the list can be a very intensive operation
-            // owing to its use of ride_has_any_track_elements().
-            // This makes sure it's only refreshed every 64 ticks.
-            if (!(gCurrentRealTimeTicks & 0x3f))
-            {
-                RefreshList();
-            }
```

Comment:
No idea why this was in the `OnResize` function. I think we might have invoked it in `OnPrepareDraw` before recent refactors, in which case it _used_ to work at least.

Moved the logic to `OnPeriodicUpdate` instead.